rust/Cargo.toml.in

```diff
@@ -69,6 +67,8 @@ time = "=0.3.20"
 
 suricata-derive = { path = "./derive", version = "@PACKAGE_VERSION@" }
 
+suricata-lua-sys = { git = "https://github.com/jasonish/suricata-lua-sys", version = "0.1.0-alpha.1" }
```

Comment:
think we can accept this for now, but we'd need a ticket to move this into the OISF github